tests/fixtures/odata_response_missing_content_geometry.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
Normally, we test cases like this though actual requests/responses that we record into VCR cassettes. I can see that storing a JSON body with exactly the kind of response you want to test is very convenient. But when `_parse_odata_response` changes in the future, this will be a pain to trace back.

Reply:
Thanks for the clarification. Outdated though since I now removed the testcase.